targets/XPlatCppSdk/source/build/Windows/XPlatCppWindows.vcxproj.filters.ejs

```diff
@@ -26,7 +26,7 @@
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabPlatformMacros.h">
       <Filter>Header Files\playfab</Filter>
     </ClInclude>
-    <ClInclude Include="$(SdkSourceDir)\include\playfab\OneDSHttpPlugin.h">
+    <ClInclude Include="$(SdkSourceDir)\include\playfab\OneDSCurlHttpPlugin.h">
```

Comment:
We can probably remove them from the filter as well since we arent building it.

Reply:
Good catch!

---
In reply to: [245173781](https://github.com/PlayFab/SDKGenerator/pull/369#discussion_r245173781) [](ancestors = 245173781)